File: Data Structures & Algorithms/single-number/submission-1.js
class Solution {
    /**
     * @param {number[]} nums
     * @return {number}
     */
    singleNumber(nums) {
        if (nums.length == 1) return nums[0];
        let z = 0;
        for (let i = 0; i < nums.length; i++){
            z = z ^ nums[i];
        }
        return z
    }
}

File: Data Structures & Algorithms/minimum-size-subarray-sum/submission-1.py
class Solution:
    def minSubArrayLen(self, target: int, nums: List[int]) -> int:
        left = 0
        ret = 1000000001
        s = 0
        for r in range(len(nums)):
            s += nums[r]
            while s >= target:
                s -= nums[left]
                ret = min(ret, r - left + 1)
                left += 1
        if ret > 1000000000:
            return 0
        return ret